packages/synapse-react-client/src/lib/style/bootstrap4_backports/_base-import.scss

```
@@ -7,7 +7,6 @@
   //initial bootstrap config based on Synapse design
   $primary: SRC.$primary-action-color;
   $secondary: SRC.$secondary-action-color;
-  $border-radius: 0.2rem;
   $btn-box-shadow: SRC.$box-shadow-soft;
```

If you look in `node_modules/bootstrap/scss` you can find some of the variables that bootstrap uses that we are overriding.

In bootstrap's `_buttons.scss` it looks like they use `$btn-border-radius`. Does this override work and also let us remove some of the extra styling below?

```suggestion
  $btn-border-radius: SRC.$button-border-radius;
  $btn-box-shadow: SRC.$box-shadow-soft;
```

---

Thanks for the tip! 

The override does work for standard sized buttons, so I would be able to remove the styling [here](https://github.com/Sage-Bionetworks/synapse-web-monorepo/pull/126/files#r1164474037). However, since bootstrap scales the border-radius depending on the button size, I'd also add an override for `$btn-border-radius-lg` to prevent the border-radius scaling on large buttons (`size="lg"`) and another override for small buttons.

Alternately, I could leave the styling in the `.btn` block, since that overrides the border-radius for all button sizes. Which do you think is the better option?